test/common/formatter/substitution_format_string_test.cc

```diff
@@ -127,9 +128,8 @@ TEST_F(SubstitutionFormatStringUtilsTest,
 )EOF";
   TestUtility::loadFromYaml(yaml, config_);
 
-  EXPECT_THROW_WITH_MESSAGE(SubstitutionFormatStringUtils::fromProtoConfig(config_, context_),
-                            EnvoyException,
-                            "Failed to create command parser: envoy.formatter.FailFormatter");
+  EXPECT_EQ(SubstitutionFormatStringUtils::fromProtoConfig(config_, context_).status().message(),
```

Comment:
Consider using
```
EXPECT_THAT(SubstitutionFormatStringUtils::fromProtoConfig(config_, context_), HasStatusMessage(...));
```
(from `test/test_common/status_utility.h`)
(or using `HasStatus` to also validate the status code)